experimental/iterators/lib/Conversion/IteratorsToLLVM/IteratorsToLLVM.cpp

```diff
@@ -196,6 +229,530 @@ struct PrintOpLowering : public ConversionPattern {
   }
 };
 
+//===----------------------------------------------------------------------===//
+// Helpers for creating Open/Next/Close functions
+//===----------------------------------------------------------------------===//
+
+/// Creates a new function at the parent module of originalOp with the given
+/// types and name and initializes the function body with a first block.
+FuncOp createOpenNextCloseFunc(Operation *originalOp, RewriterBase &rewriter,
```

Comment:
`static` qualifier, here and everywhere we don't want the function to escape.

Reply:
Thanks! Done in next commit.